multinode_integration_tests/tests/blockchain_interaction_test.rs

```diff
@@ -28,7 +28,7 @@ use node_lib::sub_lib::wallet::Wallet;
 #[test]
```

Comment:
I hope you made sure on your computer that this test passes. I remember you trying so I believe you did. I'm saying so because it's a big weakness of this test that it does its job only if the developer doesn't forget to run the multinode tests in his dev environment. More than 90% of time we have no chances of an alarm ringing from this test as it sits down here just in silence and happy. :)